deliverable/1/deliverable_1.md

```diff
@@ -0,0 +1,41 @@
+# Deliverable 1
+
+Handout: https://www.utsc.utoronto.ca/~atafliovich/cscd01/project/deliverable1.pdf
+
+## Development Process
+
+The process we chose is an heavily modified "waterfall" process incorporating elements from test-driven development (especially useful for bug fixing and large unfamiliar projects) and from Agile (for work estimation and some tracking). We also incorporated elements from Kanban to minimize clutter on our Trello board and to ensure that tasks are not left abandoned for unreasonable periods, especially those upon which others are dependent.
+
+To see a full explanation of the process we designed (and the reasons why) see [here](process.md).
+
+## Architecture Analysis
+
+Documentation [source available here](https://github.com/matplotlib/matplotlib/tree/master/doc) and [hosted here](https://matplotlib.org/3.1.1/contents.html).
+
+Primarily, matplotlib is a library, whose `import`able elements are under the [matplotlib](https://github.com/matplotlib/matplotlib/tree/master/lib/matplotlib) module.
+
+### The 3-tiered Architecture
+
+The current architecture of matplotlib is based around creating, rendering, and updating `Figure` objects. It consists of three stacked up layers. The **encapsulation** of Matplotlib is in such a way that each layer in the stack is only aware of and interacts with lower levels which results in an appropriate distribution of complexity. Each layer is responsible for a particular functionality such as event-based interactions, visual component abstraction and creation, or stateful user interaction ![Top level Diagram](./img/UML_Top_Level.svg)

```

Comment:
The `!` mark before Top Level Diagram

Reply:
This is just syntax for the image, but I have included a break between the paragraph and the image.